Title: 137847 – gtkmozembed on gnome-python-extras doesn't work against firefox
URL: https://bugs.gentoo.org/show_bug.cgi?id=137847

Go to: 
		Gentoo Home
Documentation
Forums
Lists
Bugs
Planet
Store
Wiki
Get Gentoo!





Gentoo's Bugzilla – Bug 137847
gtkmozembed on gnome-python-extras doesn't work against firefox
Last modified: 2006-11-12 20:47:44 UTC node [vulture]


Home
| New–[Ex]
| Browse
| Search
| Privacy Policy

| 





[?]
| Reports

| 
Requests

| 
Help


| 
New Account


| 
Log In





[x]



| 
Forgot Password

Login:




[x]













Bug 137847 
      - gtkmozembed on gnome-python-extras doesn't work against firefox


Summary:
gtkmozembed on gnome-python-extras doesn't work against firefox
    








Status:
    

RESOLVED
          WORKSFORME
      






Alias:


        None
    





Product:

Gentoo Linux




Classification:

Unclassified




Component:

[OLD] GNOME

  (show other bugs)



Hardware:

All
        Linux
      







Importance:
      
High
       normal
      


Assignee:

Gentoo Linux Gnome Desktop Team








URL:







Whiteboard:




Keywords:








Depends on:







Blocks:









 





      Reported:
    
2006-06-24 08:10 UTC by AlbertoRuiz





      Modified:
    
2006-11-12 20:47 UTC
      (History)
    




          CC List:
        

0 
          users
        








See Also:






Package list:







Runtime testing required:

---























      Attachments
    



Add an attachment
        (proposed patch, testcase, etc.)
    








Note
              You need to
              log in
              before you can comment on or make changes to this bug.
            

















Description


AlbertoRuiz





          2006-06-24 08:10:33 UTC
        

gnome-python-extras package has a problem with pygtkmozembed, if I try to compile it using USE="firefoz -mozilla" then it raises Segmentation Fault when I use make an instance of the MozEmbed widget, if I recompile gnome-python-extras against mozilla, instead firefox, it works ok.


Portage 2.1 (default-linux/x86/2006.0, gcc-3.4.4, glibc-2.3.5-r2, 2.6.16-gentoo-r9 i686)
=================================================================
System uname: 2.6.16-gentoo-r9 i686 Intel(R) Pentium(R) M processor 1.73GHz
Gentoo Base System version 1.6.14
dev-lang/python:     2.4.2
dev-python/pycrypto: 2.0.1-r5
dev-util/ccache:     [Not Present]
dev-util/confcache:  [Not Present]
sys-apps/sandbox:    1.2.12
sys-devel/autoconf:  2.13, 2.59-r6
sys-devel/automake:  1.4_p6, 1.5, 1.6.3, 1.7.9-r1, 1.8.5-r3, 1.9.6-r1
sys-devel/binutils:  2.16.1
sys-devel/gcc-config: 1.3.12-r6
sys-devel/libtool:   1.5.22
virtual/os-headers:  2.6.11-r2
ACCEPT_KEYWORDS="x86 ~x86"
AUTOCLEAN="yes"
CBUILD="i386-pc-linux-gnu"
CFLAGS="-O2 -mtune=i686 -pipe"
CHOST="i386-pc-linux-gnu"
CONFIG_PROTECT="/etc"
CONFIG_PROTECT_MASK="/etc/env.d /etc/gconf"
CXXFLAGS="-O2 -mtune=i686 -pipe"
DISTDIR="/usr/portage/distfiles"
FEATURES="autoconfig distlocks metadata-transfer sandbox sfperms strict"
GENTOO_MIRRORS="http://distfiles.gentoo.org http://distro.ibiblio.org/pub/linux/distributions/gentoo"
MAKEOPTS="-j2"
PKGDIR="/usr/portage/packages"
PORTAGE_RSYNC_OPTS="--recursive --links --safe-links --perms --times --compress --force --whole-file --delete --delete-after --stats --timeout=180 --exclude='/distfiles' --exclude='/local' --exclude='/packages'"
PORTAGE_TMPDIR="/var/tmp"
PORTDIR="/usr/portage"
SYNC="rsync://rsync.europe.gentoo.org/gentoo-portage"
USE="x86 X alsa apache2 apm avi berkdb bitmap-fonts bzip2 cairo cdr cli crypt cups dbus dri dvd eds emboss encode esd firefox foomaticdb fortran gdbm gif gnome gpm gstreamer gtk gtk2 hal imlib ipv6 isdnlog jpeg kde libg++ libwww mad mikmod motif mp3 mpeg ncurses nls nptl ogg opengl oss pam pcre pdflib perl png pppd python quicktime readline reflection sdl session spell spl ssl tcpd theora tiff truetype truetype-fonts type1-fonts udev unicode usb vorbis xml xorg xv zlib elibc_glibc input_devices_keyboard input_devices_mouse input_devices_synaptics kernel_linux userland_GNU video_cards_nvidia video_cards_nv video_cards_gforce video_cards_geforce video_cards_ati video_cards_radeon video_cards_fbdev video_cards_vesa"
Unset:  CTARGET, EMERGE_DEFAULT_OPTS, INSTALL_MASK, LANG, LC_ALL, LDFLAGS, LINGUAS, PORTAGE_RSYNC_EXTRA_OPTS, PORTDIR_OVERLAY




Comment 1


Jakub Moc (RETIRED)






          2006-06-24 12:07:40 UTC
        

Please, don't restrict bugs without any reason.




Comment 2


Luis Medinas (RETIRED)






          2006-11-12 20:47:44 UTC
        

on 2.16 gnome-python-desktop and gnome-python-extras works fine with firefox (note that now it uses seamonkey instead of mozilla)

please test with 2.16 if you still have the same problem please reopen.









Format For Printing
 - XML
 - Clone This Bug
 - Clone In The Same 
                        Product
 - Top of page 







Home
| New–[Ex]
| Browse
| Search
| Privacy Policy

| 





[?]
| Reports

| 
Requests

| 
Help


| 
New Account


| 
Log In





[x]



| 
Forgot Password

Login:




[x]

